components/google_cloud/README.md

```diff
@@ -5,3 +5,5 @@ services from their own applications. Services that can be accessed include
 Google Cloud Storage, Google Cloud Datastore, Google Cloud Functions, and
 Google Cloud Pub/Sub. With the Google Cloud API, developers can build a variety
 of applications that take advantage of Google Cloud services.
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Reply:
Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).